test/integration_tests/test_models.py

```diff
@@ -0,0 +1,67 @@
+import torch
+import torchtext
+
+from ..common.assets import get_asset_path
+from ..common.torchtext_test_case import TorchtextTestCase
+
+
+class TestModels(TorchtextTestCase):
+    def test_roberta_base(self):
```

Comment:
I think this can be written in much compact manner with parameterization.

Reply:
Added a followup item. Let me do this in a separate PR